Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,55 @@
+OEM Component Subsystem- Auto Light
+------
+
+## The function of the Auto Light Subsystem
+The Master microcontroller will receive the output of the Nissan Leaf AutoLight in this part of the subsystem. Furthermore, the Master microcontroller will detect if the headlight is off, on, blinking, and high and low beams.
+
+### Specification & constraints
```

Comment:
The constraints need to state the requirements that must be met for the system to work. The only one here that is specific says the voltage (what voltage??) must not exceed 5 volts. I have no idea what is meant by that.

By looking at your schematic, it is clear that you have more design constraints in your head than what you communicated. Good! Now, you simply need to be sure that you capture those here. 

As an example, you designed the system to read in 12 volts from the lights. Yet, you have not captured that as a constraint or required function. Be sure that you specifically capture the requirements and constraints of the system.